### PR TITLE
OpenID iOS support

### PR DESCRIPTION
**Pull request by Andrew-Tan**
Added OpenID iOS support. For now, since retrieving connection config is not yet supported, all configs are hard coded in source. Maybe this request should be merged after this problem is solved.

### CODE REVIEW

File: src/ios/OpenIDAuth.m [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+@property (atomic, retain) CDVPlugin* mPlugin;
+@end
+
+@implementation OpenIDAuth
+
+static OpenIDAuth *sharedInstance;
+
+
+
+// TODO: read from connection config so that these setting are not direcly in here.
```
[the remark this answers is not in the document]
Comment (Andrew-Tan) [reply src/ios/OpenIDAuth.m]: For now I'll leave it as it is. I'll do the change together with the change with server later.

File: src/ios/OpenIDAuth.m [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                                                           [self setAuthState:nil];
+                                                           authResultCallback(nil, error);
+                                                       }
+                                                   }];
+}
+
+/*! @brief Logs a message to stdout and the textfield.
+ @param format The format string and arguments.
+ */
+- (void)logMessage:(NSString *)format, ... NS_FORMAT_FUNCTION(1,2) {
```
[the remark this answers is not in the document]
Comment (Andrew-Tan) [reply src/ios/OpenIDAuth.m]: Is this going to display the error message to the UI?

[the remark this answers is not in the document]
Comment (shankari) [reply src/ios/OpenIDAuth.m]: no. Unless you have `showUI` = true

**Comment by shankari** [comment]
LGTM!

**Pull request merged**